code/game/objects/structures/crates_lockers/closets/secure/medical.dm

```diff
@@ -64,7 +63,6 @@
 	new /obj/item/megaphone/command(src)
 	new /obj/item/defibrillator/compact/loaded(src)
 	new /obj/item/clothing/gloves/color/latex/nitrile(src)
-	new /obj/item/storage/belt/medical(src)

```

Comment:
I use both duffel and medbelt nearly every round as CMO.
They're available elsewhere, but these aren't just random items that sit unused.

Reply:
it's easily dispensable from the medivend, much like the RCD is dispensable from engivend : angry: